src/components/CreateNoteCard/CreateNoteCard.jsx

```diff
@@ -3,15 +3,21 @@ import "./create-note-card.css";
 import { useAuthContext } from "../../contexts/authContext";
 import { addNote, updateNote } from "../../util/util";
 import { useNoteContext } from "../../contexts/noteContext";
+import { useColorContext } from "../../contexts/colorContext";
```

Comment:
You can use index.js file in your context folder and then import all contexts from there

Reply:
thanks will implement that